### PR TITLE
Cdp testcase

### PR DESCRIPTION
**Pull request by pratikasr**

### CODE REVIEW

File: x/cdp/keeper/cdp_test.go [hunk 1]
```diff
@@ -0,0 +1,278 @@
...
+func (suite *CdpTestSuite) TestAddCdp() {
+
+	addrs, _ := sdk.AccAddressFromBech32("abc")
+
+
+	cdp := types.NewCDP(types.DefaultIndex, addrs,sdk.NewCoin("xrp", sdk.NewInt(200000000)), "xrp-a", sdk.NewCoin("usdx", sdk.NewInt(500000000)))
+	suite.keeper.SetCDP(suite.ctx, cdp)
+
+	params := types.CollateralParam{
+		CollateralDenom:  "xrp",
```
Comment (prashantkrv2020) [inline x/cdp/keeper/cdp_test.go]: dont use xrp as example

File: x/cdp/keeper/cdp_test.go [hunk 2]
```diff
@@ -0,0 +1,278 @@
...
+		DebtDenom:        "usdx",
```
Comment (prashantkrv2020) [inline x/cdp/keeper/cdp_test.go]: use uscx as example for stable coin

File: x/cdp/keeper/cdp_test.go [hunk 3]
```diff
@@ -0,0 +1,278 @@
...
+		Type:             "xrp-a",
+		LiquidationRatio: sdk.Dec{},
+	}
+	cparams := [] types.CollateralParam{params}
+	suite.keeper.SetParams(suite.ctx,types.Params{cparams})
+
+	ak := suite.app.GetAccountKeeper()
```
Comment (prashantkrv2020) [inline x/cdp/keeper/cdp_test.go]: use full names accountKeeper instead of ak

File: x/cdp/keeper/cdp_test.go [hunk 4]
```diff
@@ -0,0 +1,278 @@
...
+}
+
+func (suite *CdpTestSuite) TestKeeper_VerifyCollateralizationRatio() {
+
+	suite.keeper.SetParams(suite.ctx, types.Params{})
+	params := types.CollateralParam{
+		CollateralDenom:  "cmdx",
+		DebtDenom:        "usdx",
+		Type:             "cmdx-a",
+		LiquidationRatio: sdk.Dec{},
```
Comment (prashantkrv2020) [inline x/cdp/keeper/cdp_test.go]: give sample liquidation ratio as well

File: x/cdp/keeper/keeper.go [hunk 5]
```diff
@@ -36,7 +36,7 @@ func NewKeeper(
 
 	return &Keeper{
 		cdc:           cdc,
-		storeKey:      storeKey,
+		StoreKey:      storeKey,
```
Comment (prashantkrv2020) [inline x/cdp/keeper/keeper.go]: revert this

File: x/cdp/types/params_test.go [hunk 6]
```diff
@@ -0,0 +1,15 @@
+package types
+
+import (
+	"github.com/cosmos/cosmos-sdk/x/params/types"
+	"reflect"
+	"testing"
+)
+
+func TestParamKeyTable(t *testing.T) {
+	a := reflect.TypeOf(ParamKeyTable())
```
Comment (prashantkrv2020) [inline x/cdp/types/params_test.go]: avoid single character names like a and b. use meaningful names

**Review by prashantkrv2020 — DISMISSED** [review]

**Comment by pratikasr** [comment]
Will resolve error and reopen again

**Pull request closed without merging**